### PR TITLE
Installer for release versions

### PR DESCRIPTION
**Pull request by BenediktMKuehne**
**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
---
* adds EMBA installation option without git

**What is the current behavior?** (You can also link to an open issue here)
---
installer relies on exclusively git functions for some stuff

**What is the new behavior (if this is a feature change)? If possible add a screenshot.**
---
Installation should work from tarball/zip release

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
---
nope

**Other information**:
---
#180 
#181

### CODE REVIEW

File: installer.sh [hunk 1]
```diff
@@ -535,43 +560,50 @@ uninstall (){
...
-  # delete/uninstall submodules
+  
   # emba
   if [[ -d ./emba/external ]]; then
     rm -r ./emba/external/
   fi
-  # all submodules
-  if [[ ${REFORCE} -eq 1 ]]; then
-    sudo -u "${SUDO_USER:-${USER}}" git submodule status
+  if [[ "${NO_GIT}" -eq 1 && ${REFORCE} -eq 0 ]]; then
```
Comment (m-1-k-3) [inline installer.sh]: Please ensure consistent quoting

```
if [[ "${NO_GIT}" -eq 1 && "${REFORCE}" -eq 0 ]]; then
```

File: installer.sh [hunk 2]
```diff
@@ -535,43 +560,50 @@ uninstall (){
...
+    # simple delete emba
+    rm -RIv ./emba
   else
-    if [[ $(sudo -u "${SUDO_USER:-${USER}}" git submodule foreach git status --porcelain --untracked-files=no) ]]; then
-      echo -e "[!!]${RED}""${BOLD}""Submodule changes detected - please commit them...otherwise they will be lost""${NC}"
-      read -p "If you know what you are doing you can press any key to continue ..." -n1 -s -r
+    # delete/uninstall submodules
+    if [[ ${REFORCE} -eq 1 ]]; then
```
Comment (m-1-k-3) [inline installer.sh]: please quote

**Review by m-1-k-3 — CHANGES_REQUESTED** [review]
sorry for the delay ... I have missed this PR

**Review by HoxhaEndri — APPROVED** [review]
I tested the installation and everything works fine. Just add the quotations like Mike wrote.

**Review by m-1-k-3 — APPROVED** [review]

**Pull request merged**